### PR TITLE
Allen&Heath.dLiveDirector version 1.88

### PR DESCRIPTION
**Pull request by JudahZF**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20291)

### CODE REVIEW

File: manifests/a/Allen&Heath/dLiveDirector/1.88/Allen&Heath.dLiveDirector.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Allen&Heath.dLiveDirector
+PackageVersion: 1.88
+PackageLocale: en-US
+Publisher: Allen & Heath
+PackageName: dLive Director
+License: https://www.allen-heath.com/terms/general-eula/
+ShortDescription: dLive Director 1.88 Setup
```
Comment (KaranKad) [inline manifests/a/Allen&Heath/dLiveDirector/1.88/Allen&Heath.dLiveDirector.locale.en-US.yaml:9]:
```suggestion
ShortDescription: dLive Director is a multi-platform editor and control software for dLive.
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

File: manifests/a/Allen&Heath/dLiveDirector/1.88/Allen&Heath.dLiveDirector.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Allen&Heath.dLiveDirector
+PackageVersion: 1.88
```
Comment (ItzLevvie) [inline manifests/a/Allen&Heath/dLiveDirector/1.88/Allen&Heath.dLiveDirector.locale.en-US.yaml:4]:
```suggestion
PackageVersion: V1.88 - Rev. 64094
```

File: manifests/a/Allen&Heath/dLiveDirector/1.88/Allen&Heath.dLiveDirector.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Allen&Heath.dLiveDirector
+PackageVersion: 1.88
```
Comment (ItzLevvie) [inline manifests/a/Allen&Heath/dLiveDirector/1.88/Allen&Heath.dLiveDirector.installer.yaml:4]:
```suggestion
PackageVersion: V1.88 - Rev. 64094
```

File: manifests/a/Allen&Heath/dLiveDirector/1.88/Allen&Heath.dLiveDirector.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Allen&Heath.dLiveDirector
+PackageVersion: 1.88
```
Comment (ItzLevvie) [inline manifests/a/Allen&Heath/dLiveDirector/1.88/Allen&Heath.dLiveDirector.yaml:4]:
```suggestion
PackageVersion: V1.88 - Rev. 64094
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
You'll have to update the version folder too.

**Pull request closed without merging**